### ISSUE TITLE
src/test/resources/import.sql not loaded

### ISSUE THREAD

**Issue by loicmathieu** [quarkusio/quarkus#6545]
**Describe the bug**
In **test** mode, when we put the `import.sql` script inside `src/test/resources` it is not taken into account by Hibernate but it is taken into account when it is inside `src/main/resources`.

**Expected behavior**
In **test** we could be able to put the import script inside `src/test/resources`.

**Actual behavior**
In **test** we must put the import script inside `src/main/resources`.

**To Reproduce**
1. Create an application with the Hibernate extension
2. Put an `import.sql` script inside `src/main/resources`.
3. Create a test
4. Launch the test via `mvn clean test`

**Configuration**
```properties
# Add your application.properties here, if applicable.
# configure your datasource
quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:test
quarkus.datasource.driver=org.h2.Driver

# drop and create the database at startup (use `update` to only update the schema)
quarkus.hibernate-orm.database.generation = drop-and-create
```

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Linux 5.3.0-26-generic #28-Ubuntu SMP Wed Dec 18 05:37:46 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
 - Output of `java -version`: openjdk version "11.0.5" 2019-10-15
 - GraalVM version (if different from Java): not applicable
 - Quarkus version or git rev: 1.1.0.Final

**Additional context**
(Add any other context about the problem here.)

**Comment by geoand** [comment]
@loicmathieu I don't agree with this. 
Why do you believe that `src/test/resources` should be taken into account when running the dev mode?

Nothing else from `src/test` is taken into account in dev mode, so why would this case be an exception?

cc @gsmet

**Comment by loicmathieu** [comment]
@geoand Sorry, this is **test** mode of course. I update my issue.

**Comment by geoand** [comment]
OK, thanks!

Then yeah, this sounds like a legitimate issue

**Comment by gastaldi** [comment]
I created a test and I can't reproduce the problem you're describing. If I add a `import.sql` script in src/test/resources, that file is used when my @QuarkusTest is run. What am I missing?

**Comment by geoand** [comment]
@loicmathieu if you have a reproducer ready, that would be great

**Comment by gastaldi** [comment]
This is my test case: https://github.com/gastaldi/quarkus/commit/3e0b74feed43c5e14986990492e357f8be52b759

**Comment by loicmathieu** [comment]
@gastaldi there is an existing import.sql inside src/main/resources in the jpa integration test so this is not the same test case.
I triple check and I still have the issue in my project. I will try to provide a reproducer

**Comment by snajfi** [comment]
Hello. I have same problem. I made small repo with example and description. Please feel free to look. 
[QuarkusBugReproduction6545](https://github.com/snajfi/QuarkusBugReproduction6545)

**Comment by loicmathieu** [comment]
Thanks @snajfi to provides a reproducer, I was too busy (and a little too lazy ;) ) to create one yet !

**Comment by jonathanvila** [comment]
In my case I am redefining the file name to be used : 
%test.quarkus.hibernate-orm.sql-load-script = import-test.sql

But apparently I need to put that import-test.sql file in src/main/resources not in src/test/resources

I dont know if this is on purpose.

**Comment by snajfi** [comment]
Hello. I made test and this issue is reproducible also in last stable version (1.2.0.Final).

**Comment by gastaldi** [comment]
Can someone test against master? I believe #7052 might have fixed this

**Comment by gastaldi** [comment]
Fixed in 1.3.0.Final
